### PR TITLE
Scrolling is not working with prev/next buttons

### LINKED ISSUE #8291: Scrolling is not working with prev/next buttons
Compare with scrolling when changing groups via group-menu:
The UI should scroll to the start of the configuration area.
It worked before als with prev/next buttons but now it is somehow broken (sometimes it works only with every second click).

### PR DESCRIPTION
**Pull request by Larisa-Staroverova**

### CODE REVIEW

File: feature-libs/product/configurators/common/src/components/config-previous-next-buttons/config-previous-next-buttons.component.spec.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ import { Observable, of } from 'rxjs';
 import { take } from 'rxjs/operators';
 import { TestScheduler } from 'rxjs/testing';
 import { ConfigPreviousNextButtonsComponent } from './config-previous-next-buttons.component';
+import { ConfigUtilsService } from '@spartacus/product/configurators/common';
```
Comment (ChristophHi) [inline feature-libs/product/configurators/common/src/components/config-previous-next-buttons/config-previous-next-buttons.component.spec.ts]: Please consume via path here!

File: feature-libs/product/configurators/common/src/components/service/config-utils.service.ts [hunk 2]
```diff
@@ -53,4 +61,50 @@ export class ConfigUtilsService {
...
     return localAssembledValues;
   }
+
+  /**
+   * Verifies whether the HTML element is in the viewport.
+   *
+   * @param {Element} element - HTML element
+   * @return {boolean} Returns 'true' if the HTML element is in the viewport, otherwise 'false'
+   */
+  private isInViewport(element: Element): boolean {
```
Comment (ChristophHi) [inline feature-libs/product/configurators/common/src/components/service/config-utils.service.ts]: Please chose 'protected'!

File: feature-libs/product/configurators/common/src/components/service/config-utils.service.ts [hunk 3]
```diff
@@ -53,4 +61,50 @@ export class ConfigUtilsService {
...
+        (window.innerWidth || document.documentElement.clientWidth)
+    );
+  }
+
+  /**
+   * Scrolls to the corresponding HTML element.
+   *
+   * @param {Element | HTMLElement} element - HTML element
+   */
+  private scroll(element: Element | HTMLElement): void {
```
Comment (ChristophHi) [inline feature-libs/product/configurators/common/src/components/service/config-utils.service.ts]: protected

**Review by ChristophHi — CHANGES_REQUESTED** [review]

File: feature-libs/product/configurators/common/src/components/config-previous-next-buttons/config-previous-next-buttons.component.spec.ts (hunk 1 above)
Comment (Larisa-Staroverova) [reply feature-libs/product/configurators/common/src/components/config-previous-next-buttons/config-previous-next-buttons.component.spec.ts]: You are absolutely right. I will adjust it

File: feature-libs/product/configurators/common/src/components/service/config-utils.service.ts (hunk 2 above)
Comment (Larisa-Staroverova) [reply feature-libs/product/configurators/common/src/components/service/config-utils.service.ts]: Sure

File: feature-libs/product/configurators/common/src/components/service/config-utils.service.ts (hunk 3 above)
Comment (Larisa-Staroverova) [reply feature-libs/product/configurators/common/src/components/service/config-utils.service.ts]: Sure

**Review by ChristophHi — APPROVED** [review]

**Pull request merged**